### PR TITLE
fixed typo

### PR DESCRIPTION
**Pull request by atinfinity**
# This pullrequest changes

I fixed the following typo in opencv.

- `lenght` -> `length`
- `Lenght` -> `Length`

### CODE REVIEW

File: apps/interactive-calibration/defaultConfig.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <opencv_storage>
 <charuco_dict>0</charuco_dict>
-<charuco_square_lenght>200</charuco_square_lenght>
+<charuco_square_length>200</charuco_square_length>
```
Comment (dkurt) [inline apps/interactive-calibration/defaultConfig.xml]: not sure about changing a config file because it can lead to incompatibility issues

Comment (alalek) [reply apps/interactive-calibration/defaultConfig.xml]: I believe this compatibility code can handle migration:
```
if (readFromNode(reader["charuco_square_lenght"], mCapParams.charucoSquareLength))
    std::cout << "DEPRECATION: Parameter 'charuco_square_lenght' has been deprecated (typo). Use 'charuco_square_length' instead." << std::endl;
readFromNode(reader["charuco_square_length"], mCapParams.charucoSquareLength);
```

Comment (atinfinity) [reply apps/interactive-calibration/defaultConfig.xml]: @alalek @dkurt Thank you for your advice.

> I believe this compatibility code can handle migration

I'll fix this point.

Comment (asmorkalov) [reply apps/interactive-calibration/defaultConfig.xml]: @atinfinity do you have progress on this?

Comment (atinfinity) [reply apps/interactive-calibration/defaultConfig.xml]: @asmorkalov Sorry for the late reply. I'll try to do this weekend.

Comment (atinfinity) [reply apps/interactive-calibration/defaultConfig.xml]: @asmorkalov I fixed this point. https://github.com/opencv/opencv/pull/16445/commits/db87044f4ec7d43aeb442bf0d48e032c1c25f814

**Pull request merged**